src/app/app.module.ts

```diff
@@ -2,7 +2,7 @@
 
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import { FormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
```

Comment:
Why are you adding ReactiveFormsModule? I do not see any feature being added that requires this.

Reply:
Before I had resolved the membership page submission I was considering adding ReactiveForms and will remove this addition.